cypress/components/NotificationsDrawer.cy.tsx

```diff
@@ -0,0 +1,244 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import DrawerPanel from '../../src/components/NotificationsDrawer/DrawerPanel';
```

Comment:
Hitting an error here. Does this exist as declared?

Reply:
Hey! This is the component we are refactoring to this repo (and why Bryan and I added cypress tests here). PR for that #661